modules/auxiliary/scanner/http/ms15_034_ulonglongadd.rb

```diff
@@ -0,0 +1,91 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'MS15-034 HTTP Protocol Stack Request Handling Vulnerability',
+      'Description'    => %q{
+        This module will check if your hosts are vulnerable to CVE-2015-1635 (MS15-034). A
+        vulnerability in the HTTP Protocol stack (HTTP.sys) that could result in arbitrary code
+        execution. Please note this module could potentially cause a denail-of-service against
+        the servers you're testing.
+      },
+      'Author'         =>
+        [
+          'Bill Finlayson', # He did all the work (see the pastebin code), twitter: @hectorh56193716
+          'sinn3r'          # MSF version of bill's work
+        ],
+      'References'     =>
+        [
+          ['CVE', '2015-1635'],
+          ['MSB', 'MS15-034'],
+          ['URL', 'http://pastebin.com/ypURDPc4']
+        ],
+      'License'        => MSF_LICENSE
+    ))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The base path', '/'])
```

Comment:
Hmm iisstart.htm doesn't actually dos for me...

How about this, I will leave this info in the description suggesting what files you could use, and then we'll leave it to the user to decide?